cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -115,7 +115,7 @@ library
     io-classes >=1.4.1,
     memory,
     memory-pool,
-    mempack,
+    mempack >=0.2,
```

Comment:
This constraint is not needed due to CPP that makes it compatible with mempack-0.1 as well. Such approach usually simplifies the transition